src/components/PriceBar/TileColorBoard.tsx

```diff
@@ -1,9 +1,10 @@
 import { Center, Flex, Text } from "@chakra-ui/react";
 import React, { useEffect, useState, useMemo } from "react";
 import { useStoreSelector } from "@/store/hooks";
-import { IPriceCalculation } from "../../interfaces/priceCalculation";
+import { IPriceCalculation } from "@/interfaces/priceCalculation";
 import { useGetPriceQuery } from "@/redux/api/priceApi";
 import priceFormat from "@/utils/priceFormat";
+import BudgetBoard from "./BudgetBoard";
```

Comment:
Is this BudgetBoard on the TileColorBoard or on the index of PriceBar? I can't find any relationship that BudgetBoard should be the child of TileColorBoard.